### ISSUE TITLE
so many words in the QuickStart.rst that i cannot figure out what is necessary

### ISSUE THREAD

**Issue by zycskyblue** [robotframework/QuickStartGuide#13]

**Comment by jcgsykes** [comment]
Agreed - Mingling documentation and code like this is very unhelpful, as this isn't likely to replicate a real case. Is is possible to produce a realistic "Hello World" style test case so that people can actually learn something?

**Comment by haonit** [comment]
Agreed too. It's not friendly to newbie.
